store/postgres/src/mapping.rs

```diff
@@ -0,0 +1,1141 @@
+use diesel::connection::SimpleConnection;
+use diesel::{debug_query, PgConnection, RunQueryDsl};
+use graphql_parser::query as q;
+use graphql_parser::schema as s;
+use inflector::Inflector;
+use std::cmp::PartialOrd;
+use std::collections::{HashMap, HashSet};
+use std::fmt;
+use std::rc::Rc;
+use std::str::FromStr;
+
+use crate::mapping_sql::{
+    ClampRangeQuery, ConflictingEntityQuery, CopyQuery, EntityData, FilterQuery, FindQuery,
+    InsertQuery, RevertClampQuery, RevertRemoveQuery,
+};
+use graph::prelude::{
+    format_err, Entity, EntityChange, EntityChangeOperation, EntityFilter, EntityKey,
+    QueryExecutionError, StoreError, StoreEvent, SubgraphDeploymentId, ValueType,
+};
+
+use crate::block_range::BlockNumber;
+use crate::entities::STRING_PREFIX_SIZE;
+
+trait AsDdl {
```

Comment:
This trait seems over-abstracted, as we discussed it can not be a trait and be an inherent method `as_ddl` that returns a string, the `fmt` that abstracts over the writer doesn't seem necessary.

Reply:
Made it all inherent methods. Commit 3508afb5

Reply:
Looks like that commit is not pushed yet.

Reply:
Sorry, yes, forgot to push. It's there now.